posts/models.py

```diff
@@ -1,7 +1,14 @@
 from django.db import models
+from users.models import User
 
 
 class Post(models.Model):
     text = models.TextField()
     created_at = models.DateTimeField(auto_now_add=True)
     approved = models.BooleanField(default=False)
+    user = models.ForeignKey(User, on_delete=models.CASCADE, default=1)
```

Comment:
this field should be required and there should be no default here. I can't imagine a situation how a post could be created without a user.